server/service/src/sync/sync_status/test.rs

```diff
@@ -110,7 +110,7 @@ async fn sync_status() {
                 .map(|i| {
                     inline_init(|r: &mut LocationRow| {
                         r.id = i.to_string();
-                        r.store_id = mock_store_a().id;
+                        r.store_id = mock_store_b().id;
```

Comment:
Store needs to be on the right `sync_site` or it won't be in the outgoing records list...
(Result of adding store_id to Location sync change logs)